internal/circonus/alerting.go

```diff
@@ -17,414 +17,370 @@ import (
+            }
+        ]        
+    },
+    "deployment_glitches": {
+        "filter": "and(deployment:*)",
+        "lookup_key": "k8s_health_deploys",
+        "metric_name": "deployment_generation_delta",
+        "metric_type": "numeric",
+        "name": "Kubernetes Deployment Glitches ({cluster_name})",
+        "rules": [
+			{
+				"wait": 0,
+				"severity": 0,
+				"windowing_duration": 300,
+				"windowing_min_duration": 0,
+				"criteria": "on absence",
+				"windowing_function": null,
+				"value": 900
+        	},
+            {
+                "criteria": "max value",
+                "severity": 1,
+                "wait": 0,
+                "windowing_duration": 300,
+                "value": "0"
+            },
+            {
+                "criteria": "min value",
+                "severity": 1,
+                "wait": 0,
+                "windowing_duration": 300,
+                "value": "0"
+            }
+        ]        
+    },
+    "daemonsets_not_ready": {
+        "filter": "and(daemonset:*)",
+        "lookup_key": "k8s_health_daemonsets",
+        "metric_name": "daemonset_scheduled_delta",
+        "metric_type": "numeric",
+        "name": "Kubernetes DaemonSets Not Ready ({cluster_name})",
+        "rules": [
+			{
+				"wait": 0,
+				"severity": 0,
+				"windowing_duration": 300,
+				"windowing_min_duration": 0,
+				"criteria": "on absence",
+				"windowing_function": null,
+				"value": 900
+        	},
+            {
+                "criteria": "max value",
+                "severity": 1,
+                "wait": 0,
+                "windowing_duration": 300,
+                "value": "0"
+            },
+            {
+                "criteria": "min value",
+                "severity": 1,
+                "wait": 0,
+                "windowing_duration": 300,
+                "value": "0"
+            }
+        ]
+    },
+    "statefulsets_not_ready": {
+        "filter": "and(statefulset:*)",
+        "lookup_key": "k8s_health_statefulsets",
+        "metric_name": "statefulset_replica_delta",
+        "metric_type": "numeric",
+        "name": "Kubernetes StatefulSets Not Ready ({cluster_name})",
+        "rules": [
+			{
+				"wait": 0,
+				"severity": 0,
+				"windowing_duration": 300,
+				"windowing_min_duration": 0,
+				"criteria": "on absence",
+				"windowing_function": null,
+				"value": 900
+        	},
+            {
+                "criteria": "max value",
+                "severity": 1,
+                "wait": 0,
+                "windowing_duration": 300,
+                "value": "0"
+            },
+            {
+                "criteria": "min value",
+                "severity": 1,
+                "wait": 0,
+                "windowing_duration": 300,
+                "value": "0"
+            }
+        ]
+    }
 }
+`
 
-func defaultRules() (map[string]apiclient.RuleSet, error) {
-	defaultRuleSetsData := []byte(`
+const defaultRuleSetsStr118 = `
```

Comment:
In my latest research I discovered that one metric change didn't happen until 1.20, so I'm pegging my new dashboard to a v1.20 minimum, so you should do so also, I think.

Reply:
resolved in [d71873a](https://github.com/circonus-labs/circonus-kubernetes-agent/pull/91/commits/d71873af18c6bbca5d8fb9c2066daf06d888b9f6)